src/reducers/television.reducers.js

```diff
@@ -0,0 +1,1625 @@
+export default function () {
+
+	var id = 0;
+
+	var data = [
+		{
+			"category": "Smart",
+			"type": "LED TV",
+			"screen Size": "32 inch (80 cm)",
+			"price": 30000,
+			"screen Resolution": "1366 x 768",
+			"aspect Ratio": "16:9",
+			"supported Audio Formats": [
+				"MP3", "AAC", "WMA"
+			],
+			"connectivity": {
+				"USB ports": 2,
+				"HDMI ports": 2,
+				"internet": true,
+			},
+			"HD": "Full HD",
+			"weight": 4.6,
+			"color": "Black",
+			"model": "Samsung32J5500AK",
+			"brand": "Samsung",
+			"Power Consmption": 90,
```

Comment:
Power Consumption

Reply:
@priyanka271990 Its always in watt, you need to static string of "W" into it.

Reply:
ok. But the spelling of consumption is wrong.

Reply:
Oh good catch. Just correct it and create the pr.